tests/std/tests/GH_000108_deque_incomplete_value_types/test.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <deque>
+
+using namespace std;
+
+void test_gh_000108();

```

Comment:
Can we put each test into a respective function with a proper name that explains what it is doing?